scripts/azureml-assets/azureml/assets/get_tokens.py

```diff
@@ -41,22 +49,32 @@ def get_tokens(input_dirs: List[Path],
         if not isinstance(path, AzureBlobstoreAssetPath):
             continue
 
-        account_name = model_config.path.storage_name
-        container_name = model_config.path.container_name
+        add_token_info(path, json_info)
 
-        if container_name in json_info[account_name]:
-            continue
+    with open(json_output_path, 'w') as json_token_file:
+        json.dump(json_info, json_token_file)
 
-        _ = model_config.path.get_uri(token_expiration=timedelta(days=1))
-        token = model_config.path.token
 
-        if token is not None and len(token) == 0:
-            token = None
+def add_token_info(path: AzureBlobstoreAssetPath, json_info: dict):
```

Comment:
```suggestion
def add_token_info(path: AzureBlobstoreAssetPath, json_info: defaultdict(dict)):
```
Update for arg type in comment as well